.github/workflows/run-integ-tests.yaml

```diff
@@ -84,14 +125,20 @@ jobs:
         app_to_test: ${{ fromJson(needs.list-apps.outputs.to_test) }}
         mode: [fresh, upgrade]
         exclude:
-          - # issue $618
+          - # issue #618
```

Comment:
nit: These reference tickets on the old repo.

Reply:
yeah, I wasn't really sure what to do with these in general. Should I refer to it as klotho-history#618? Do we want to advertise that in-the-code-ly the existence of that repo?

We can't move issues from a private repo to a public one (I tried), or else I'd have just moved it over as part of this.

Reply:
Probably have to just move them the old fashioned way - good 'ol C-c C-v 😄